README.md

```diff
@@ -33,7 +33,7 @@ To disable cloning and installing, use:
 
 Run
 
-```$ docker run --name ardrive-cli-bats --rm --init -td --mount type=tmpfs,destination=/home/node/tmp ardrive-bats-docker && sleep 20 ```
+```$ docker run --name ardrive-cli-bats --rm --init -tdi --mount type=tmpfs,destination=/home/node/tmp ardrive-bats-docker && sleep 20 ```
```

Comment:
What is the behavior of Docker when it runs with `-d` + `-i`? What's the deal with running the docker without the interactive flag in this case?